helix-view/src/editor.rs

```diff
@@ -132,8 +196,8 @@ pub struct Config {
     pub cursorline: bool,
     /// Highlight the columns cursors are currently on. Defaults to false.
     pub cursorcolumn: bool,
-    /// Gutters. Default ["diagnostics", "line-numbers"]
-    pub gutters: Vec<GutterType>,
+    /// Gutters. Default { layout: ["diagnostics", "spacer", "line-numbers"] }
```

Comment:
I think we can just remove the docs here on the default since the type is getting more complicated